impeller/renderer/backend/gles/texture_gles.cc

```diff
@@ -362,7 +362,7 @@ static std::optional<GLenum> ToRenderBufferFormat(PixelFormat format) {
   switch (format) {
     case PixelFormat::kB8G8R8A8UNormInt:
     case PixelFormat::kR8G8B8A8UNormInt:
-      return GL_RGBA4;
+      return GL_RGBA8;
```

Comment:
This really kept me guessing for a while, thankfully ANGLE can be easily debugged.